lib/token.h

```diff
@@ -972,6 +977,7 @@ class CPPCHECKLIB Token {
         fIsLiteral              = (1 << 21),
         fIsTemplateArg          = (1 << 22),
         fIsAttributeNodiscard   = (1 << 23), // __attribute__ ((warn_unused_result)), [[nodiscard]]
+        fIsCommentedOut			= (1 << 24),
     };
```

Comment:
This will conflict with https://github.com/danmar/cppcheck/pull/1353.

Whichever commit goes in second will need to resolve this.